### PR TITLE
doc: scaleway_compute_instance_ip

### PR DESCRIPTION
**Pull request by QuentinBrosse**

### CODE REVIEW

File: website/docs/r/compute_instance_ip.html.markdown [hunk 1]
```diff
@@ -3,41 +3,44 @@ layout: "scaleway"
...
 # scaleway_compute_instance_ip
 
-Creates and manages Scaleway compute instance IPs.
+Creates and manages Scaleway Compute Instance IPs.
 
 ## Example Usage
 
 ```hcl
-resource "scaleway_compute_instance_ip" "test" {}
+resource "scaleway_compute_instance_ip" "ip" {}
```
Comment (kindermoumoute) [inline website/docs/r/compute_instance_ip.html.markdown]: nit: "myip"?

File: website/docs/r/compute_instance_ip.html.markdown [hunk 2]
```diff
@@ -3,41 +3,44 @@ layout: "scaleway"
...
 
 The following arguments are supported:
 
-* `project_id` - (Optional) The id of the project you want to attach this resource to.
-* `reverse` - (Optional) The reverse dns for this IP.
-* `zone` - (Optional) The zone you want to attach this resource to.
+- `address` - (Computed) The IP address.
+- `reverse` - (Optional) The reverse DNS for this IP.
+- `project_id` - (Optional) The ID of the project you want to attach this resource to. If it is not provided, the provider `project_id` is used.
+- `zone` - (Optional) The zone you want to attach this resource to. If it is not provided, the provider `zone` is used.
```
Comment (kindermoumoute) [inline website/docs/r/compute_instance_ip.html.markdown]: nit: link to the developer website?

File: website/docs/r/compute_instance_ip.html.markdown [hunk 3]
```diff
@@ -3,41 +3,44 @@ layout: "scaleway"
...
 
-* `project_id` - The id of the project your resource is attached to.
-* `reverse` - The reverse dns for this IP.
-* `server_id` - The id of the server this resource is attached to.
-* `zone` - The zone your resource is attached to.
+- `id` - The ID of the IP.
+- `address` - The IP address.
+- `reverse` - The reverse DNS for this IP.
+- `server_id` - The ID of the server this resource is attached to.
+- `project_id` - The ID of the project you want to attach this resource to.
```
Comment (kindermoumoute) [inline website/docs/r/compute_instance_ip.html.markdown]: precise somewhere the organization_id is a valid project_id

File: website/docs/r/compute_instance_ip.html.markdown [hunk 4]
```diff
@@ -3,41 +3,44 @@ layout: "scaleway"
...
+## Arguments Reference
 
 The following arguments are supported:
 
-* `project_id` - (Optional) The id of the project you want to attach this resource to.
-* `reverse` - (Optional) The reverse dns for this IP.
-* `zone` - (Optional) The zone you want to attach this resource to.
+- `reverse` - (Optional) The reverse DNS for this IP.
+- `server_id` - (Optional) The ID of the server you want to attach this resource to.
+- `project_id` - (Optional) The ID of the project you want to attach this resource to. If it is not provided, the provider `project_id` is used.
```
Comment (kindermoumoute) [inline website/docs/r/compute_instance_ip.html.markdown]: The ID of the project the IP is associated with.

File: website/docs/r/compute_instance_ip.html.markdown [hunk 5]
```diff
@@ -3,41 +3,44 @@ layout: "scaleway"
...
-## Argument Reference
+## Arguments Reference
 
 The following arguments are supported:
 
-* `project_id` - (Optional) The id of the project you want to attach this resource to.
-* `reverse` - (Optional) The reverse dns for this IP.
-* `zone` - (Optional) The zone you want to attach this resource to.
+- `reverse` - (Optional) The reverse DNS for this IP.
+- `server_id` - (Optional) The ID of the server you want to attach this resource to.
```
Comment (kindermoumoute) [inline website/docs/r/compute_instance_ip.html.markdown]: The ID of the server you want to attach this IP to.

File: website/docs/r/compute_instance_ip.html.markdown [hunk 6]
```diff
@@ -3,41 +3,44 @@ layout: "scaleway"
...
+- `project_id` - (Optional) The ID of the project you want to attach this resource to. If it is not provided, the provider `project_id` is used.
+- `zone` - (Optional) The [zone](https://developers.scaleway.com/en/quickstart/#zone-definition) you want to attach this resource to. If it is not provided, the provider `zone` is used.
```
Comment (kindermoumoute) [inline website/docs/r/compute_instance_ip.html.markdown]: The zone in which the IP should be reserved.

File: website/docs/r/compute_instance_ip.html.markdown [hunk 7]
```diff
@@ -3,41 +3,44 @@ layout: "scaleway"
...
 The following attributes are exported:
 
-* `project_id` - The id of the project your resource is attached to.
-* `reverse` - The reverse dns for this IP.
-* `server_id` - The id of the server this resource is attached to.
-* `zone` - The zone your resource is attached to.
+- `id` - The ID of the IP.
+- `address` - The IP address.
+- `reverse` - The reverse DNS for this IP.
+- `server_id` - The ID of the server this resource is attached to.
```
Comment (kindermoumoute) [inline website/docs/r/compute_instance_ip.html.markdown]: up

File: website/docs/r/compute_instance_ip.html.markdown [hunk 8]
```diff
@@ -3,41 +3,44 @@ layout: "scaleway"
...
+- `project_id` - The ID of the project you want to attach this resource to.
```
Comment (kindermoumoute) [inline website/docs/r/compute_instance_ip.html.markdown]: up

File: website/docs/r/compute_instance_ip.html.markdown [hunk 9]
```diff
@@ -3,41 +3,44 @@ layout: "scaleway"
...
+- `zone` - The [zone](https://developers.scaleway.com/en/quickstart/#zone-definition) your resource is attached to.
```
Comment (kindermoumoute) [inline website/docs/r/compute_instance_ip.html.markdown]: up

File: website/docs/r/compute_instance_ip.html.markdown [hunk 10]
```diff
@@ -3,41 +3,44 @@ layout: "scaleway"
...
 
 ## Import
 
-Instances can be imported using the `zone/id`, e.g.
+Instances can be imported using the `{zone}/{id}`, e.g.
 
-```
-$ terraform import scaleway_compute_instance_ip.test fr-par-1/11111111-1111-1111-1111-111111111111
+```bash
+$ terraform import scaleway_compute_instance_ip.server_ip fr-par-1/11111111-1111-1111-1111-111111111111
```
Comment (kindermoumoute) [inline website/docs/r/compute_instance_ip.html.markdown]: Question, should we remove the locality from the ID so only the ID is require to import a resource? I sasw this behavior was possible with the Import [StateFunc](https://github.com/hashicorp/terraform/blob/master/helper/schema/resource_importer.go#L34)

Comment (QuentinBrosse) [reply website/docs/r/compute_instance_ip.html.markdown]: How are we going to find the right locality if it is not in the ID?

File: website/docs/r/compute_instance_ip.html.markdown (hunk 7 above)
Comment (QuentinBrosse) [reply website/docs/r/compute_instance_ip.html.markdown]: I follow this order: https://github.com/terraform-providers/terraform-provider-scaleway/blob/cd5978fcd435d7b277c98decb93b0f26cf67b7c2/scaleway/resource_compute_instance_ip.go#L18-L36

File: website/docs/r/compute_instance_ip.html.markdown (hunk 8 above)
Comment (QuentinBrosse) [reply website/docs/r/compute_instance_ip.html.markdown]: I follow this order: https://github.com/terraform-providers/terraform-provider-scaleway/blob/cd5978fcd435d7b277c98decb93b0f26cf67b7c2/scaleway/resource_compute_instance_ip.go#L18-L36

File: website/docs/r/compute_instance_ip.html.markdown (hunk 9 above)
Comment (QuentinBrosse) [reply website/docs/r/compute_instance_ip.html.markdown]: I follow this order: https://github.com/terraform-providers/terraform-provider-scaleway/blob/cd5978fcd435d7b277c98decb93b0f26cf67b7c2/scaleway/resource_compute_instance_ip.go#L18-L36

**Pull request merged**